pyroute2/netlink/rtnl/req.py

```diff
@@ -192,6 +234,29 @@ def __setitem__(self, key, value):
                                                      value['type']))
                     dict.__setitem__(self, 'encap',
                                      self.encap_header(value))
+                # human-friendly form:
+                #
+                # 'encap': {'type': 'seg6',
+                #           'mode': 'encap'
+                #           'segs':'2000::5,2000::6'}
```

Comment:
btw, it should work also as `'segs': ['2000::5', '2000::6']`, shouldn't it?

Reply:
yes, it works.